src/ui/components/timelabel/TimeLabel.tsx

```diff
@@ -30,6 +35,7 @@ export const DEFAULT_TIME_LABEL_STYLE: TextStyle = {
 export const TimeLabel = (props: TimeLabelProps) => {
   const currentTime = useCurrentTime();
   const duration = useDuration();
-  const { showDuration, style } = props;
-  return <StaticTimeLabel showDuration={showDuration} time={currentTime} duration={duration} style={[DEFAULT_TIME_LABEL_STYLE, style]} />;
+  const { showDuration, style, scrubTime } = props;
+  const expectedSeekTarget = scrubTime || currentTime
```

Comment:
This doesn't handle scrubbing to `0`. Use `??` instead.
```suggestion
  const expectedSeekTarget = scrubTime ?? currentTime
```